spack/packages/dla-future-fortran/package.py

```diff
@@ -37,7 +37,7 @@ class DlaFutureFortran(CMakePackage):
 
     depends_on("dla-future@0.4.1:0.5 +scalapack", when="@0.1.0")
     depends_on("dla-future@0.6.0: +scalapack", when="@0.2.0:")
-    depends_on("dla-future@master: +scalapack", when="@main")
+    depends_on("dla-future@0.7.0: +scalapack", when="@main")
```

Comment:
Should this be 
```suggestion
    depends_on("dla-future@0.7.0: +scalapack", when="@0.3:")
```
already now, or would you updated that after the release?

Reply:
I was thinking of doing this after the release, when `0.3.0` is added to the Spack package (post-release PR). But I don't mind either way, maybe it's good to do it here already.

Reply:
Either way, but I definitely don't disapprove of you updating it now!